### ISSUE TITLE
Cant install

### ISSUE THREAD

**Issue by Bodengriller** [felipec/wd-persistent-ssh#2]
Hi,
I love your idea - but I can't install the app on OS5.26.300 on my MyCloud Mirror :(

![image](https://github.com/felipec/wd-persistent-ssh/assets/50927649/ab9a8e4c-b8dd-4c05-b424-3756ec7faace)

**Comment by felipec** [comment]
I don't know. The support from WD is pretty shitty.

As explained in the README, I used the SDK called `My_Cloud_OS_5_NAS_App_Guide_04292021.zip`, I got the address from this forum question: [Whare are the OS5 SDK tools?](https://community.wd.com/t/whare-are-the-os5-sdk-tools/266486).

There's probably a new SDK, but if there is, I've no idea where it is or how to get it. WD just does not help.

If you manage to get the binary `MyCloudOS5_mksapkg` from a newer SDK I can try to recompile this package and see if it works.

But other than that I don't know what I can do. My package is dead simple.
